src/TLuaInterpreter.h

```diff
@@ -194,6 +194,7 @@ class TLuaInterpreter : public QThread
     static int clearSpecialExits(lua_State*);
     static int solveRoomCollisions(lua_State*);
     static int setGridMode(lua_State* L);
+    static int getGridMode(lua_State* L);
```

Comment:
The `L` is not needed here, is it? :slightly_frowning_face:

Reply:
Doesn't hurt

Reply:
Fair enough. :man_shrugging: